Immense.RemoteControl.Server/Hubs/DesktopHub.cs

```diff
@@ -242,21 +242,21 @@ public Task SendMessageToViewer(string viewerId, string message)
 
     public async Task SendDesktopStream(IAsyncEnumerable<byte[]> stream, Guid streamId)
     {
-        var session = _streamCache.GetOrAdd(streamId, key => new StreamSignaler(streamId));
-        session.DesktopConnectionId = Context.ConnectionId;
+        using var signaler = _streamCache.GetOrAdd(streamId, key => new StreamSignaler(streamId));
+        signaler.DesktopConnectionId = Context.ConnectionId;
 
         try
         {
-            session.Stream = stream;
-            session.ReadySignal.Release();
-            await session.EndSignal.WaitAsync(TimeSpan.FromHours(8));
+            signaler.Stream = stream;
+            signaler.ReadySignal.Release();
+
+            await _hubEvents.NotifyRemoteControlStarted(SessionInfo);
+            await signaler.EndSignal.WaitAsync(TimeSpan.FromHours(8));
```

Comment:
Yeah.  I just picked an arbitrary amount of time that would be longer than the vast majority of sessions.  The concern being that someone would leave their browser open and go on vacation.  If the user's computer is always-on, the remote control session could be running in the background for days, eating up bandwidth.

Ultimately, I need to add an input idle timer, so if the viewer isn't interacting with the page after X timespan, it will prompt them.  Like, "Are you still there? (30 seconds until auto-disconnect.)"